recipe/meta.yaml

```diff
@@ -11,27 +11,36 @@ source:
 
 build:
   noarch: python
+  skip: true # [py<37]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
     - poetry-core >=1.0.0
-    - python >=3.7
+    - python
+    - wheel
   run:
-    - python >=3.7
+    - py >=1.11.0
+    - python
     - werkzeug >=2.0.0
 
 test:
+  imports:
+    - coverage
   source_files:
     - tests
   requires:
     - pip
     - pytest >=7.1.3
+    - pytest-cov
     - requests >=2.28.1
+    - types-requests >=2.28.9
+    - pre-commit 3.2.1
```

Comment:
Upstream i using 2.20 (https://github.com/csernazs/pytest-httpserver/blob/master/pyproject.toml#L74), any reason to pin to 3.2.1 specifically?

Reply:
This probably also applies to the other comment but I was told that we should pick "stable" dependencies as in "the ones that we support" over necessarily upgrading everything to the ones upstream uses. That is part of our "value add" and I think leans in on the LTS nomenclature.

Not saying it's right or wrong, but that was my understanding about how we operate as a team/company.

Reply:
I'm not sure I follow that logic. We have `pre-commit 2.20` and it's not an upgrade compared to `3.2.1`. If upstream required `^2.20.0`, we should follow those requirements.

However, I'm not sure all test requirement are really necessary. `pytest-cov` is not needed for a simple `pytest` call and I suspect `pre-commit` may not be needed either to run the tests.

Reply:
`conda search pre-commit` is what you're looking for.

The import errors probably come from you importing `coverage`. Looking at the upstream feedstock, I would expect you to actually import `pytest_httpserver`.